package.json

```diff
@@ -77,6 +77,9 @@
     "ngc-webpack": "3.0.0",
     "node-sass": "4.5.3",
     "open": "0.0.5",
+    "@pact-foundation/karma-pact": "^2.1.1",
```

Comment:
For consistency with our other dependencies, let's use exact versions here (no caret).